docs/usage-examples/runCommand.txt

```diff
@@ -0,0 +1,52 @@
+.. _laravel-run-command-usage:
+
+=============
+Run a Command
+=============
+
+You can run a command directly on a database by calling the ``command()``
+method on a database connection instance.
+
+To run a command, call the ``command()`` method and specify the database command
+document inside the method call. This document includes the command name and
+value.
+
+Example
+-------
+
+This usage example performs the following actions:
+
+- Creates a database connection instance to access the ``sample_mflix`` database
+- Specifies a command to retrieve a list of collections in the ``sample_mflix`` database
+- Prints the name of each ``sample_mflix`` collection
```

Comment:
Issue:

There's no reference to "sample_mflix" in the code example.

Suggestion:

I think it could be good to either describe as the database configured for your "mongodb" connection or to add some information before this list that mentions that the mongodb connection is set to sample_mflix.